### ISSUE TITLE
"Image" terminology no longer accurate

### ISSUE THREAD

**Issue by sanmadjack** [shish/shimmie2#752]
Shimmie used to be just images, but now it is not. Despite this, media is called an image in all of these places:

1. UI text
2. Class and variable names
3. Database tables

Obviously this is a mostly cosmetic issue, but replacing each item has it's merits. I think at minimum #1 should be handled so that the user doesn't get messages calling a mp3 an image. Obviously renaming classes and variables has the potential to create many bugs, and if we do it we should update everything so we can get things consistent going forward. Renaming database tables/fields has legacy implications, and the potential to break third-party extensions.

Personally, I think it's worth it, but obviously it's not my call.

Potential new terms:

1. Item
2. Post
3. Datum? I only really had two ideas.

A variation of this, which would really only affect #1, is to allow file type handlers to declare a noun for their items. Then a video could be called a video, an image an image, etc.

**Comment by shish** [comment]
Agreed with those three items in pretty much that order - UI text being most important and easiest to change, database tables being least important and most difficult (renaming a table while keeping foreign key constraints etc across multiple supported databases T_T)

I have been thinking that "Post" would make more sense - we're already using eg /post/view/1234 in URLs

**Comment by sanmadjack** [comment]
I'd agree. Should I just start changing gui text, and maybe class/variable names?

**Comment by shish** [comment]
Sounds like a good plan :)

Also a 4th thing that should also be done carefully - updating URLs, to avoid breaking people's bookmarks. Perhaps this won't be a problem because the #1 thing for people to bookmark is /post/view pages and that *already* uses the word "post", but just in case...
